src/domain/domainClasses.js

```diff
@@ -0,0 +1,30 @@
+// TODO: Discuss if we should export every class or only the Route class.
+
```

Comment:
Why is this file in this branch and in this pull request? It will collide with my current implementation of these classes. But it will be easy to fix, I guess

Reply:
+1 should not be here